python_modules/dagster/dagster/core/execution/context/output.py

```diff
@@ -372,6 +381,83 @@ def get_output_identifier(self) -> List[str]:
 
         return identifier
 
+    def log_event(
+        self, event: Union[AssetObservation, AssetMaterialization, Materialization]
+    ) -> None:
+        """Log an AssetMaterialization or AssetObservation from within the body of an io manager's `handle_output` method.
+
+        Events logged with this method will appear in the list of DagsterEvents, as well as the event log.
```

Comment:
Whats "the list of DagsterEvents"?

Reply:
If we're executing via the python layer, there is a concrete set of dagsterevents, but if we're being honest mentioning the event log is probably enough